dependencies/src/test/java/com/google/cloud/tools/opensource/dependencies/DependencyGraphBuilderTest.java

```diff
@@ -370,6 +370,30 @@ public void testConfigureAdditionalMavenRepositories_addingGoogleAndroidReposito
     assertNotNull(graph);
   }
 
+  @Test
+  public void testConfigureAdditionalMavenRepositories_resolvingMultipleArtifacts() {
+    // To verify the effect of the test, you need to cleanup your local Maven repository
```

Comment:
This is iffy. The test should not depend on the state of the local maven repository.

Reply:
Moving this to DependencyGraphIntegrationTest